odkx-src/suitcase-intro.rst

```diff
@@ -113,34 +113,35 @@ Downloading from the server
 Uploading to the server
   When uploading, you will need to specify the *Version* which by default is *2*. By default ODK-X Suitcase assumes the upload field to be
   a :file:`Upload` directory where the ODK-X Suitcase jar file is located to change it click on the :guilabel:`...` button.
-  
+
   To Upload files to ODK-X Cloud Endpoint, you need to lay out the files and folders in the correct file structure 
   which is described in details in the :doc:`config-file-structure`.
-  
+
   Your upload directory should look similar to the *config* directory and contain
   subdirectories *assets* and/or *tables* as shown in the :doc:`config-file-structure`.
   An example for the same can be found `here <https://github.com/odk-x/app-designer/tree/master/app/config>`_. 
-  
+
   Then modify the :guilabel:`Upload` field to that file path by clicking on the :guilabel:`...` button, and then press :guilabel:`Upload`.
 
   Screenshot of upload tab:
 
   .. image:: /img/suitcase-intro/upload_tab.png
    :width: 600
 
-  .. note::
-
-      Suitcase GUI supports only Uploading of files and not Updating or Modifying, check out :ref:`Suitcase CLI  <suitcase-cli-update>`
-
   .. _suitcase-gui-reset:
-  
+
 Resetting the server
-  The :guilabel:`Reset` button can be found under the :guilabel:`Clear` tab. Clicking it will reset the the server after a warning and a confirmation.
+  The :guilabel:`Reset` button can be found under the :guilabel:`Clear` tab. Clicking :guilabel:`Reset` will reset the the server after a warning and a confirmation.
+
+  Screenshot of Clear Tab:
```

Comment:
I think this should have the same :guilabel:'Clear' formatting. Tab should be lower case.